modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -88,15 +104,16 @@ The certificate must be set with a passphrase and a common name (CN) that specif
 **** If the CN does not contain a wildcard, the endpoint takes the form `{common-name}/{app-name}`.
 
 ... Specify a value for *Key File*. This is the PEM formatted file that contains the private key for the certificate.
-... If the *Key Passcode* field is displayed after uploading a PEM key file, enter the word or phrase that protects the private key.
-... Optionally, specify a value for *CA Path Certificate File (Optional)*. The CA path contains the intermediary and root certificates that provide the path (chain of trust) from the certificate to the root. When you provide an entry in this field, the path is sent along with the certificate during the TLS handshake.
-... Select *Deploy*. The *Key Passcode* field is blanked out for security reasons. You can still review public certificate details. If you upload a new key file, this field is again enabled.
+... (Optional) If the *Key Passcode* field is displayed after uploading a PEM key file, enter the word or phrase that protects the private key.
+... Specify a value for *CA Path Certificate File (Optional)*. The CA path contains the intermediary and root certificates that provide the path (chain of trust) from the certificate to the root. The file is downloadable from the CA that signed your certificate. When you provide an entry in this field, the Runtime Fabric load balancer at every connection sends both the certificate and the path to the client. Many clients require the server to send the CA path so that the certificate can be validated.
+... Use the *Details* button to view certificate details, including domains, expiration, and the other configuration settings.
+... Select *Deploy* after specifying any needed security policies or advanced options. The *Key Passcode* field is blanked out for security reasons. You can still review public certificate details. If you upload a new key file, this field is again enabled.
```

Comment:
```suggestion
... Select *Deploy*.
```
If the audience needs to "specify any needed security policies or advance options," then we should have steps for that before deployment.